frontend/src/components/pages/Dropoff/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Container, Text } from "@chakra-ui/react";
+import React, { useContext, useState } from "react";
+import { useHistory } from "react-router-dom";
+
+import DonorAPIClient from "../../../APIClients/DonorAPIClient";
+import SchedulingAPIClient from "../../../APIClients/SchedulingAPIClient";
+import * as Routes from "../../../constants/Routes";
+import AuthContext from "../../../contexts/AuthContext";
+import Header from "../../common/Header";
+import DropoffCard from "./components/DropoffCard";
+
+const Dropoff = (): JSX.Element => {
+  const { authenticatedUser } = useContext(AuthContext);
+  const [schedule, setSchedule] = useState<any>(null);
+  const history = useHistory();
+
+  if (!authenticatedUser) {
+    history.push(Routes.LOGIN_PAGE);
+  }
+
+  React.useEffect(() => {
+    const getSchedules = async () => {
+      const response = await SchedulingAPIClient.getScheduleByDonorId("2");
+      setSchedule(response);
+    };
+
+    getSchedules();
+  }, [authenticatedUser]);
+
+  console.log("USER: ", authenticatedUser);
```

Comment:
do we need these logs?

Reply:
this is still work in progress!! going to remove that `useEffect` and add a donor context